phoenix-core-server/src/main/java/org/apache/phoenix/coprocessor/BaseRegionScanner.java

```diff
@@ -42,9 +42,7 @@ public boolean isFilterDone() {
     public abstract boolean next(List<Cell> results) throws IOException;
 
     @Override
-    public boolean next(List<Cell> result, ScannerContext scannerContext) throws IOException {
```

Comment:
Why is that ?
I can see this implemented in child classes.

Reply:
Declared next with scannerContext as abstract to make sure all child classes implement it.